src/sonic-build-hooks/hooks/dpkg

```diff
@@ -3,7 +3,10 @@
 . /usr/local/share/buildinfo/scripts/buildinfo_base.sh
 REAL_COMMAND=$(get_command dpkg)
 COMMAND_INFO="Locked by command: $REAL_COMMAND $@"
-lock_result=$(acquire_apt_installation_lock "$COMMAND_INFO" )
+if ! printenv LOCKED > /dev/null || [[ "$(printenv LOCKED)" != "y" ]];then
```

Comment:
```suggestion
if [ "$LOCKED" != "y" ]; then
```

Reply:
And maybe rename to something like DPKG_HOOK_LOCKED instead of LOCKED to exclude possible conflicts.